app/build.gradle

```diff
@@ -168,9 +175,9 @@ dependencies {
     implementation 'com.squareup.picasso:picasso:2.71828'
     // Public key verification
     implementation "com.github.joshjdevl.libsodiumjni:libsodium-jni-aar:2.0.2@aar"
-    // Reactive extensions for asynchronous events
-    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
-    implementation 'io.reactivex.rxjava2:rxjava:2.2.19'
+
+    // Kotlin coroutines
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1")
```

Comment:
Let's follow the pattern we use with the rest of the dependencies, do not use parentheses:

    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1"

Also, 1.4.2 is the latest version.

Reply:
Fixed in https://github.com/eduvpn/android/pull/315/commits/6110f6631c0da912df799c53e8c31572a63e6c31